Examples/PFMExampleApp/README.md

```diff
@@ -1,10 +1,10 @@
 # PFM Example Project 
```

Comment:
Should probably change this to a more suiting title

Reply:
Will update the naming of this project together with the module and cocoapods renaming later so think this can stay like this to match the project name until then.